test/runtests.jl

```diff
@@ -20,7 +20,8 @@ include("fielddiffeq.jl")
 include("hybrid2d.jl")
 include("hybrid3d.jl")
 include("sphere.jl")
-#include("diffusion2d.jl")
+include("diffusion2d.jl")
```

Comment:
I think you can remove this line here, as `diffusion2s.jl` is already included above (I thought the order was supposed to be alphabetical, but then I actually see that later on it isn't).
By the way, thanks for the co-authored tag! I don't know why GitHub is showing that email address for me. I can let you know my email address associated to my GH account privately. Thanks!